juniper/src/http/mod.rs

```diff
@@ -217,6 +217,125 @@ where
     }
 }
 
+/// The Expected structure of the decoded JSON document for either POST or GET requests
```

Comment:
Can this be pulled out into a separate change?

Reply:
https://github.com/graphql-rust/juniper/pull/608